docs/user-guide/deployments.md

```diff
@@ -481,4 +508,4 @@ To learn more about when a pod is considered ready, see [Container Probes](/docs
 ### kubectl rolling update
 
 [Kubectl rolling update](/docs/user-guide/kubectl/kubectl_rolling-update) updates pods and replication controllers in a similar fashion.
-But deployments is recommended, since it's declarative and is server side, and has more features, such as rolling back to any previous revision even after the rolling update is done. Also, replica sets supersede replication controllers. 
+But deployments is recommended, since it's declarative and is server side, and has more features, such as rolling back to any previous revision even after the rolling update is done.
```

Comment:
nit: awkward

s/deployments is/deployments are/

s/since it's declarative and is server side, and has more features/since they are declarative, server side, and have additional features/